.github/workflows/ubuntu.yml

```diff
@@ -110,7 +111,6 @@ jobs:
         # archive the entire lib directory of Osi because we need all
         # 4 large library files and several file links to these.
         run: |
-          cd ../
```

Comment:
Why is this not necessary anymore? Isn't the default pwd the home directory?

Reply:
I added some documentation about this. The pwd is $GITHUB_WORKSPACE.